package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -382,6 +382,15 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
         },
 
         /**
+         * Returns the dashboard store on the prompt panel
+         *
+         * @return {Object}
+         */
+        getDashboard: function() {
+          return this.dashboard;
+        },
+
```

Comment:
We should encapsulate the setBlockUIOptions in the promptpanel as well. 
dashboard "feels" like a internal to prompt panel. I don't think it should be exposed.